crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/staticdata/enums/CourtType.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.justice.laa.crime.meansassessment.staticdata.enums;
+
+import com.fasterxml.jackson.annotation.JsonValue;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.stream.Stream;
+
+/**
+ * static data migrated from TOGDATA.COURT_TYPES table
+ */
+@AllArgsConstructor
+@Getter
+public enum CourtType {
+    MAGISTRATE("MAGISTRATE", "Magistrate"),
```

Comment:
I think we could just do this 
MAGISTRATE("Magistrate") 
as we have done in another part of the code base.

Reply:
Hello @atharmajeed 
Thank you for your comment. 
I appreciate that it may seem the courtType field is not needed. However, it is there for the benefit of "CROWN COURT" as that is the literal value stored in the DB (COURT_HARDSHIP_TYPES, HARDSHIP_REVIEWS tables).

@rajkrk mentioned that since this enum is not directly relevant to the initial create initial means assessment, it should be removed.

I'm happy to remove this enum and its related dataconverter if it is not immediately relevant. 

The rest of the code contains code refactoring. Mostly test coverage for another existing enum converter, FrequencyConverter.